mindsworth/Fluido-api
# #164792908 Implement the setting up GraphQL with MongoDB

#### What does this PR do?
This PR is used to set up the GraphQL server connecting to a MongoDB database and also used to configure babel.

#### Description of Task to be completed?
- [x] add all the necessary packages
- [x] bootstrapped the project
- [x] configure MongoDB to connect
- [x] configure babel

#### How should this be manually tested?
- Run git fetch on develop
- Check out to the branch `ch-configure-graphql-server-babel-mongodb-164792908`
- Run `yarn install` to install all dependencies
- Run `yarn start:dev` to start the server
- Go to your browser and browse `http://localhost:4000/graphql`
- You should see a graphiQL interface that help interact with graphQL.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#164792908](https://www.pivotaltracker.com/story/show/164792908)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
